docs/book/src/reference/metrics-reference.md

```diff
@@ -0,0 +1,22 @@
+| workqueue_retries_total                       | Counter   | Total number of retries handled by workqueue.                                                                                                                                                                                        |
+| rest_client_requests_total                    | Counter   | Number of HTTP requests, partitioned by status code, method, and host.                                                                                                                                                               |
+| controller_runtime_reconcile_total            | Counter   | Total number of reconciliations per controller.                                                                                                                                                                                      |
+| controller_runtime_reconcile_errors_total     | Counter   | Total number of reconciliation errors per controller.                                                                                                                                                                                |
+| controller_runtime_reconcile_time_seconds     | Histogram | Length of time per reconciliation per controller.                                                                                                                                                                                    |
+| controller_runtime_max_concurrent_reconciles  | Gauge     | Maximum number of concurrent reconciles per controller.                                                                                                                                                                              |
+| controller_runtime_active_workers             | Gauge     | Number of currently used workers per controller.                                                                                                                                                                                     |
+| controller_runtime_webhook_latency_seconds    | Histogram | Histogram of the latency of processing admission requests.                                                                                                                                                                           |
+| controller_runtime_webhook_requests_total     | Counter   | Total number of admission requests by HTTP status code.                                                                                                                                                                              |
+| controller_runtime_webhook_requests_in_flight | Gauge     | Current number of admission requests being served.                                                                                                                                                                                   |
```

Comment:
WDYT about we point out the source code for the metrics as well? 
Also, please ensure that we use the tag from the source code so that we will keep the history and not master in the link.

Reply:
I think it a good idea!
Was hesitating to go for it in consideration of the possible code line changes in the future.
But the tag is a nice way to resolve this 👍🏽 
..So just updated with url prefix: https://github.com/kubernetes-sigs/controller-runtime/blob/v0.11.0/...